tests/eval_files/trio91x_autofix.py

```diff
@@ -34,9 +33,9 @@ async def foo_yield():  # TRIO911: 0, "exit", Statement("yield", lineno+2)
 
 
 async def foo_if():
-    if ...:
+    if foo():
```

Comment:
Same comment about `async` `foo`

Reply:
yeah these are the same file, `eval_files` are the original files, and then the ones in in `autofix_files` are generated (and diff'd against in tests), so these two have to match.